Make.h

```diff
@@ -0,0 +1,19 @@
+//
+//  Make.h
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/8/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import <Mantle/Mantle.h>
+
+@interface Make : MTLModel <MTLJSONSerializing>
+@property (nonatomic,retain) NSNumber* makeID;
+@property (nonatomic,retain) NSString* nameEn;
+@property (nonatomic,retain) NSString* nameAr;
+@property (nonatomic,retain) NSNumber* modelsCount;
+@property (nonatomic,retain) NSNumber* enginesCount;
+@property (nonatomic,retain) NSString* imageSmallURL;
+@property (nonatomic,retain) NSString* imageLargeURL;
+@end
```

Comment:
Please consider our Objective-C Style Guide when you write your code. You can check it from here https://github.com/grapesnberries/Handbook/wiki/Objective-C-Style-Guide